### DISCUSSION TITLE
can we create new package version for aiohttp module

### DISCUSSION THREAD

**Discussion by sukruthG** [aio-libs/aiohttp#7171] [Q&A] [answered]
Hi,

We are facing a dependency issue w.r.t to the charset-normalizer module.
The issue is already addressed in this PR https://github.com/aio-libs/aiohttp/commit/cf8e25d0175f1d4874781b67c0dd466557be8e69

Our dependent issue will be resolved if we create a new package with the above fix.

Thank you

**Comment by webknjaz** [comment, marked as answer]
If somebody helps out with the stabilization of the tests (especially proxy integration) under newer Pythons (especially 3.11), it'd be fine to release aiohttp 3.9. Before that, I don't think we want to cut releases.

**Reply by Dreamsorcerer** [reply]
Shall we do a 3.9a0 release for now though?
I'd like to ensure we have some feedback time for AppKey before committing it to a public release.

**Reply by webknjaz** [reply]
That'd be `3.9.0b0`, then, I guess.

**Reply by Dreamsorcerer** [reply]
Either way. I was just thinking an alpha if we're still not comfortable with the 3.11 support (and I think there may be a few more fixes we can get in before final release too).

**Comment by Boby360** [comment]
Also looking forward to a new package version due to charset-normalizer support limitation in 3.8.3.
Would be using it with 3.10.
